mongodbatlas/resource_mongodbatlas_cluster.go

```diff
@@ -137,7 +137,7 @@ func resourceMongoDBAtlasCluster() *schema.Resource {
 			"num_shards": {
 				Type:     schema.TypeInt,
 				Optional: true,
-				Default:  1,
```

Comment:
Should we mention this as a breaking change in the Changelog? Although I understand that this is backward compatible but a change in behavior so might be worth to add to the release guide at least.

https://developer.hashicorp.com/terraform/plugin/best-practices/versioning#example-major-number-increments

Reply:
@maastha ok, I think we can put that the label on this PR